src/main/java/org/candlepin/insights/controller/InventoryController.java

```diff
@@ -267,4 +273,12 @@ public OrgInventory getInventoryForOrg(String orgId) {
     private String buildValidationMessage(ConstraintViolation<ConduitFacts> x) {
         return String.format("%s: %s: %s", x.getPropertyPath(), x.getMessage(), x.getInvalidValue());
     }
+
+    private boolean isTruncated(String factKey, String toCheck) {
+        if (toCheck != null && toCheck.endsWith("...")) {
+            log.warn("Consumer fact value was truncated. Skipping value: {}:{}", factKey, toCheck);
```

Comment:
I set is as WARN simply because this will mean that all data has not been received and it won't require debug level to see that this is happening. I'm Ok with changing it, but we should be consistent with other attribute errors as well since there are a few other places that log at the warn level. Thoughts?

Reply:
If truncated values are in fact an error condition that we (or someone else) needs to address, I agree with WARN.  Otherwise, I'd say INFO.

Reply:
I don't think anyone will be using the logs to go back and fix that, so I'll just log as info.

Reply:
@awood Updated the controller to log formatting issues at the info level.